spec/params/params.spec.ts

```diff
@@ -92,6 +104,17 @@ describe("Params value extraction", () => {
 
     const stringToBool = params.defineBoolean("A_STRING");
     expect(stringToBool.value()).to.equal(false);
+
+    const listToInt = params.defineInt("LIST");
+    expect(listToInt.value()).to.equal(0);
+  });
+
+  it("falls back on the javascript zero values in case a list param's is unparsable as string[]", () => {
+    const notAllStrings = params.defineList("BAD_LIST");
```

Comment:
no action required - but i can see this being pretty frustrating if we silent fall back to an empty list if the list isn't made up of all string elements :/.

I don't think we expect this to happen in real life since environment variable will almost always be strings anyway?